pkg/async/notifications/factory.go

```diff
@@ -103,14 +104,31 @@ func NewNotificationsProcessor(config runtimeInterfaces.NotificationsConfig, sco
 			panic(err)
 		}
 		emailer = GetEmailer(config, scope)
+		cloudProvider = common.AWS
+	case common.GCP:
+		projectID := config.GCPConfig.ProjectID
+		subscibrer := config.NotificationsProcessorConfig.QueueName
```

Comment:
nit : subscibrer -> subscriber
Also would be good if we can do some basic validation around config.GCPConfig ,ProjectId , config.NotificationsProcessorConfig and QueuName